src/main/java/com/hcl/appscan/jenkins/plugin/scanners/StaticAnalyzer.java

```diff
@@ -183,12 +188,12 @@ public boolean isScanMethod(String scanMethod) {
             return m_scanMethod.equals(scanMethod);
         }
 
-        public void validateSettings(JenkinsAuthenticationProvider authProvider, Map<String, String> properties, IProgress progress) throws AbortException {
+        public void validateScannerSettings(JenkinsAuthenticationProvider authProvider, Map<String, String> properties, IProgress progress, boolean isAppScan360) throws AbortException {
```

Comment:
Can the logic in this method be made cleaner? There are a number of places where we're checking for INCLUDE_SCA and a number of places where we check for SCAN_ID.

Reply:
Added the comments before each validation for better clarification.